source/assets/js/helpers/estaticoapp.js

```diff
@@ -65,6 +65,9 @@ class EstaticoApp {
 			};
 
 			this.initEvents = this.initEvents.concat(Module.initEvents);
+
+			// Remove duplicates from initEvents
+			this.initEvents = [...new Set(this.initEvents)];
```

Comment:
Without this addition the `initEvents` array is getting longer and longer and has several times the same values, which result in multiple inits of the modules.

(An alternative is a merge of `this.initEvents` and `Module.initEvents`.)